frontend/src/app/home/components/HomeView.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Loader from 'react-loader-spinner';
+import HomeStocksTable from '../../stocks/components/HomeStocksTable.js';
+import '../../../App.css';
+
+export default class HomeView extends React.Component {
+  render() {
+    return (
+      <div className="App">
+        {
+          (!this.props.isLoading)
+          ? <div style={styles.container}>
+              <Paper style={styles.feed}>
+                <h2>Feed</h2>
+              </Paper>
+
+              <div style={{padding: 30}}>
+                <Paper style={styles.weather}>
+                  <h2>RIT's Temperature</h2>
```

Comment:
Absolutely minor nit, but I think the .js instead of .jsx is causing github to parse the file incorrectly leading to a strange display where the GH diff here indicates that the rest of the file is an unclosed series of strings.

Reply:
I'll see if the jsx file extension will solve this.